controller/events.go

```diff
@@ -177,6 +370,13 @@ func (e *EventListener) Notify(event *ct.AppEvent) {
 			sub.Notify(event)
 		}
 	}
+	if event.AppID != "" {
```

Comment:
Nope, this is so subscribers not filtering by app get the event. Added a comment on the next line.